.github/workflows/python.yml

```diff
@@ -97,19 +97,21 @@ jobs:
         cd bindings\python
         python test.py
 
-    - name: build/install
-      if: runner.os != 'Windows'
+    - name: build/install (Linux)
+      if: runner.os == 'Linux'
       run: |
         cd bindings/python
-        # Homebrew's python "framework" sets a prefix via distutils config.
-        # --prefix conflicts with --user, so null out prefix so we have one
-        # command that works everywhere
         python3 setup.py build_ext install --user --prefix=
 
-    # TODO: we currently don't run the tests on MacOS, because something fails
-    # to install correctly. It needs to be resolved
+    - name: build/install (MacOS)
+      if: runner.os == 'macOS'
+      run: |
+        # Install to Homebrew's python site-packages. no need for --user and --prefix
+        cd bindings/python
+        python3 setup.py build_ext install --install-lib $(brew --prefix)/lib/python3.9/site-packages
```

Comment:
there's no way to ask python for this directory, rather than hard coding `python3.9`? This will break every time python is updated.

Reply:
Sorry for late reply, we could write a python 1 liner for it but idk if that would behave always the same on Mac (I don't own a Mac) hence why we are here in the first place. Or put the 3.9 on a job env and use it with brew install and here so always only update the env and everything should be fine... I am no expert so someone else could definitely weigh in on this.

Reply:
ok, thanks! Maybe @AllSeeingEyeTolledEweSew has some ideas

Reply:
Yeah, Let's see what he can conjure up. let me know if I could assist more.